src/content/app-platform/apps/observability/_index.md

```diff
@@ -0,0 +1,43 @@
+---
+linkTitle: Observability Stack
+title: Managed Observability Stack (Monitoring, Logging, Service Mesh)
+description: Overview of the Giant Swarm managed observability stack of apps, which includes monitoring, logging, and service mesh
+weight: 10
+menu:
+  main:
+    parent: app-platform-apps
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-halo
+user_questions:
+  - What app do you recommend for monitoring?
+  - What app do you recommend for logging?
+  - What app do you recommend for service mesh?
+last_review_date: 2021-08-18
+---
+
+# Giant Swarm managed Observability Stack of Apps
+
+Observability is based on three main data sources: logs, metrics and tracing*. To cover these needs Giant Swarm provides its customers with a managed observability stack. The apps included in this stack are listed below. We chose these apps based on our general principles of preferring open source over proprietary solutions, as well as simplicity, performance, and speed over complexity.
+
+**What do we mean by a ‘managed’ app?** We mean offering similar Service Level Objectives as the rest of our offering as described in [The Radical Way Giant Swarm Handles Service Level Objectives](https://www.giantswarm.io/blog/the-radical-way-giant-swarm-handles-service-level-objectives).
+
+In brief, super fast response times from engineers, 24//7 monitoring and alerting by us, and in general, having shared goals and responsibilities to keep services up and get them up when they go down.
+
+In addition, we strive to uphold our “30 Day Upgrade Promise,” our commitment to keep all managed apps within 30 days of latest upstream releases.
+
+TL;DR
+For observability, we recommend and provide these four managed apps:
+
+**1/ Prometheus for monitoring / metrics**: To get all your metrics, process, store and alert on them. Prometheus has become the de-facto standard monitoring tool in the kubernetes ecosystem.
+
+**2/ Loki for logging**: To fetch logs from containers, store and index them, and make them searchable. To install, go to [Loki README](https://github.com/giantswarm/loki-app/blob/master/README.md).
+
+In addition, we understand some customers already know and use Elasticsearch. For this reason, we also provide a managed EFK stack (Elasticsearch, Fluentd, and Kibana) for logging. To install, go to [EFK Stack README](https://github.com/giantswarm/efk-stack-app).
```

Comment:
The link strays from the other links, either change this one to link directly to the readme or change the other ones to point to just the repo.

Reply:
ok! changing this one to readme.